contrib/cirrus/setup_environment.sh

```diff
@@ -256,9 +256,9 @@ case "$TEST_FLAVOR" in
 
         # Need to re-build lists (removed during image production)
         ooe.sh apt-get -qq -y update
-        msg "Installing previously downloaded/cached packages"
+        msg "Installing docker and containerd"
         # N/B: Tests check/expect `docker info` output, and this `!= podman info`
-        ooe.sh apt-get install --yes --no-download --ignore-missing containerd.io docker-ce docker-ce-cli
+        ooe.sh apt-get install --yes containerd.io docker-ce docker-ce-cli
```

Comment:
@vrothberg the point of caching these packages and installing with `--no-download` is to prevent them from changing unexpectedly or crashing/burning due to remote repo. unavailability.  While this is a fine fix for now (un-stucking CI is important), in the future I would appreciate a `#FIXME` comment or opening an issue in https://github.com/containers/automation_images so I can fix the underlying issue.  I'll do that now, but you owe me a cookie :grin:

Reply:
Oh, and thanks for fixing this.  I do appreciate it.

Reply:
Opened https://github.com/containers/automation_images/issues/95

Reply:
Great news, thanks for digging into it, @cevich!